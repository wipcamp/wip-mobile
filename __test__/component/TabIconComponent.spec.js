/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import TabIcon from '../../components/TabIconComponent'

describe('render', () => {
    it('not focus', () => {
        const tree = renderer
            .create(
                <TabIcon
                    ios="test"
                    android="test"
                />
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })

    it('focus', () => {
        const tree = renderer
            .create(
                <TabIcon
                    fios="test"
                    fandroid="test"
                    focus
                />
            )
        expect(tree).toMatchSnapshot()
    })
})