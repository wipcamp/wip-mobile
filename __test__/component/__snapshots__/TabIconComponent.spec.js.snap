// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render focus 1`] = `
<Text
  accessible={true}
  allowFontScaling={false}
  android="test"
  ellipsizeMode="tail"
  ios="test"
  style={
    Array [
      Object {
        "color": undefined,
        "fontSize": 12,
      },
      Array [
        Object {
          "color": "#000",
          "fontSize": 30,
        },
        Object {
          "color": "#000",
          "fontSize": 30,
        },
        Object {
          "color": "#FF8214",
        },
      ],
      Object {
        "fontFamily": "Ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
    ]
  }
  virtual={undefined}
>
  ?
</Text>
`;

exports[`render not focus 1`] = `
<Text
  accessible={true}
  allowFontScaling={false}
  android="test"
  ellipsizeMode="tail"
  ios="test"
  style={
    Array [
      Object {
        "color": undefined,
        "fontSize": 12,
      },
      Array [
        Object {
          "color": "#000",
          "fontSize": 30,
        },
        Object {
          "color": "#000",
          "fontSize": 30,
        },
        Object {
          "color": "#000",
        },
      ],
      Object {
        "fontFamily": "Ionicons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
    ]
  }
  virtual={undefined}
>
  ?
</Text>
`;
